File: benchmarks/cross/progs/entity_update.js
const xs=[],ys=[],vxs=[],vys=[];for(let i=0;i<20000;i++){xs.push(i);ys.push(i);vxs.push(1.5);vys.push(-0.5);}
for(let f=0;f<200;f++)for(let j=0;j<20000;j++){xs[j]+=vxs[j];ys[j]+=vys[j];}
let t=0;for(let k=0;k<20000;k++)t+=xs[k];console.log(Math.floor(t));

File: benchmarks/cross/progs/entity_update.lua
local xs,ys,vxs,vys={},{},{},{}
for i=0,19999 do xs[i+1]=i*1.0; ys[i+1]=i*1.0; vxs[i+1]=1.5; vys[i+1]=-0.5 end
for f=1,200 do for j=1,20000 do xs[j]=xs[j]+vxs[j]; ys[j]=ys[j]+vys[j] end end
local t=0.0; for k=1,20000 do t=t+xs[k] end; print(math.floor(t))
